diff --git a/src/tools/ddsperf/ddsperf.c b/src/tools/ddsperf/ddsperf.c
index f52b7f31..569a0d21 100644
--- a/src/tools/ddsperf/ddsperf.c
+++ b/src/tools/ddsperf/ddsperf.c
@@ -797,15 +797,16 @@ static int64_t *latencystat_print (struct latencystat *y, const char *prefix, co
     ddsrt_mutex_unlock (&disc_lock);
 
     qsort (y->raw, rawcnt, sizeof (*y->raw), cmp_int64);
-    printf ("%s%s %s size %"PRIu32" mean %.3fus min %.3fus 50%% %.3fus 90%% %.3fus 99%% %.3fus max %.3fus cnt %"PRIu32"\n",
-            prefix, subprefix, ppinfo, size,
-            (double) y->sum / (double) y->cnt / 1e3,
-            (double) y->min / 1e3,
-            (double) y->raw[rawcnt - (rawcnt + 1) / 2] / 1e3,
-            (double) y->raw[rawcnt - (rawcnt + 9) / 10] / 1e3,
-            (double) y->raw[rawcnt - (rawcnt + 99) / 100] / 1e3,
-            (double) y->max / 1e3,
-            y->cnt);
+    /* printf ("%s%s %s size %"PRIu32" mean %.3fus min %.3fus 50%% %.3fus 90%% %.3fus 99%% %.3fus max %.3fus cnt %"PRIu32"\n", */
+    /*         prefix, subprefix, ppinfo, size, */
+    /*         (double) y->sum / (double) y->cnt / 1e3, */
+    /*         (double) y->min / 1e3, */
+    /*         (double) y->raw[rawcnt - (rawcnt + 1) / 2] / 1e3, */
+    /*         (double) y->raw[rawcnt - (rawcnt + 9) / 10] / 1e3, */
+    /*         (double) y->raw[rawcnt - (rawcnt + 99) / 100] / 1e3, */
+    /*         (double) y->max / 1e3, */
+    /*         y->cnt); */
+    printf ("%.9f,%.10f\n", (double) ping_intv / 1e9, (double) y->sum / (double) y->cnt / 1e3 / 2.0);
   }
   return y->raw;
 }
@@ -1617,10 +1618,12 @@ static bool print_stats (dds_time_t tref, dds_time_t tnow, dds_time_t tprev, str
     if (nrecv > 0 || substat_every_second)
     {
       const double dt = (double) (tnow - tprev);
-      printf ("%s size %"PRIu32" total %"PRIu64" lost %"PRIu64" delta %"PRIu64" lost %"PRIu64" rate %.2f kS/s %.2f Mb/s (%.2f kS/s %.2f Mb/s)\n",
-              prefix, last_size, tot_nrecv, tot_nlost, nrecv, nlost,
-              (double) nrecv * 1e6 / dt, (double) nrecv_bytes * 8 * 1e3 / dt,
-              (double) nrecv10s * 1e6 / (10 * dt), (double) nrecv10s_bytes * 8 * 1e3 / (10 * dt));
+      /* printf ("%s size %"PRIu32" total %"PRIu64" lost %"PRIu64" delta %"PRIu64" lost %"PRIu64" rate %.2f S/s %.2f Mb/s (%.2f kS/s %.2f Mb/s)\n", */
+      /*         prefix, last_size, tot_nrecv, tot_nlost, nrecv, nlost, */
+      /*         (double) nrecv * 1e9 / dt, (double) nrecv_bytes * 8 * 1e3 / dt, */
+      /*         (double) nrecv10s * 1e6 / (10 * dt), (double) nrecv10s_bytes * 8 * 1e3 / (10 * dt)); */
+      printf("%d,%.2f\n", last_size, (double)nrecv * 1e9 /dt);
+      fflush (stdout);
       output = true;
     }
 
